### ISSUE TITLE
Add remark-slug as a plugin

### ISSUE THREAD

**Issue by janine9vn** [oSTEM/conference-site#11]
# Reasoning
I'd like to add [remark-slug](https://github.com/remarkjs/remark-slug) as a plugin so that the headers on the website get auto-linked.
```md
## My Heading
```
gets turned into:
```html
<h2 id="my-heading">My Heading</h2>
```

This will allow us to add a potential Table of Contents and also provide jumplinks if need be. I feel that this necessary on pages with a lot of scrolling content like the awards page.

Another option if we don't want to use remark-slug and instead pseudo roll-our-own: [override the headings and provide our own component](https://tomekdev.com/posts/anchors-for-headings-in-mdx)

**Comment by travigd** [comment]
Yeah totally agree. Would be need if we could auto add those little clickable anchors as well to copy it.

Is this something you'd like to tackle, or just have a webdev person do it?
